README.rst

```diff
@@ -72,4 +88,4 @@ Run tests with nose::
 
 Run tests with doctest::
 
-    python -m doctest -v tailer/__init__.py
\ No newline at end of file
+    python -m doctest -v tailer/__init__.py
```

Comment:
For tail, head and follow functions you should open file in binary mode, otherwise it gives an error: 

```
__init__.py", line 58, in __init__
raise ValueError("io object must be in the binary mode")
```